bin/strata-client/src/helpers.rs

```diff
@@ -1,3 +1,5 @@
+// TODO clean up this module
+
```

Comment:
This should be moved to an issue. I don't like TODO comments in the codebase. They are easily forgotten